src/preprocessors/preprocessor.ts

```diff
@@ -26,6 +26,16 @@ export function preprocessor(code: string, options: PrettierOptions) {
     const ast = babelParser(code, parserOptions);
     const interpreter = ast.program.interpreter;
 
+    const directives: Directive[] = [];
+    traverse(ast, {
+        Directive({ node }) {
+            directives.push(node);
+
+            // Trailing comments probably shouldn't be attached to the directive
+            node.trailingComments = null;
+        },
+    });
+
```

Comment:
Can this **traverse** block not be moved into the **traverse** block ( along side with `ImportDeclaration` ) below ?

Reply:
Good catch. Fixed.